From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

This patch allows for compiling with dev-haskell/extra-1.7.10. It will break
building with earlier versions of extra.

diff -urN ghcide-1.5.0.1/src/Control/Concurrent/Strict.hs ghcide-1.5.0.1-r1/src/Control/Concurrent/Strict.hs
--- ghcide-1.5.0.1/src/Control/Concurrent/Strict.hs	2001-09-08 19:46:40.000000000 -0600
+++ ghcide-1.5.0.1-r1/src/Control/Concurrent/Strict.hs	2022-01-23 01:57:47.718909579 -0700
@@ -4,7 +4,7 @@
     ,module Control.Concurrent.Extra
     ) where
 
-import           Control.Concurrent.Extra hiding (modifyVar, modifyVar_)
+import           Control.Concurrent.Extra hiding (modifyVar, modifyVar_, modifyVar')
 import qualified Control.Concurrent.Extra as Extra
 import           Control.Exception        (evaluate)
 import           Control.Monad            (void)
